examples/pod_in_cluster_list.py

```diff
@@ -12,47 +12,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Simple example to show loading config from the cluster
-#
-# It works only from a pod. You can start an image with Python
-# (for example python:latest), exec into the pod, install the library,
-# then try out this example.
-#
-# If you get 403 errors from API server you will have to configure
-# RBAC to add the permission to list pods.
-#
-# ---
-# kind: ClusterRole
-# apiVersion: rbac.authorization.k8s.io/v1
-# metadata:
-#   name: pods-list
-# rules:
-# - apiGroups: [""]
-#   resources: ["pods"]
-#   verbs: ["list"]
-# ---
-# kind: ClusterRoleBinding
-# apiVersion: rbac.authorization.k8s.io/v1
-# metadata:
-#   name: pods-list
-# subjects:
-# - kind: ServiceAccount
-#   name: default
-#   namespace: default
-# roleRef:
-#   kind: ClusterRole
-#   name: pods-list
-#   apiGroup: rbac.authorization.k8s.io
-# ---
-#
-# Doc: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+"""
+Showcases loading the Kubernetes config from within the cluster. This script
+must be run within a pod. You can start a pod with a Python image (for example,
+`python:latest`), exec into the pod, install the library, then run this example.
+
+If you get 403 errors from the API server you will have to configure RBAC to add
+the permission to list pods:
+
+cat <<EOF | kubectl apply -f - 
```

Comment:
This is bash and requires kubectl to be installed. There are Windows users and we’re in a Python client.

Reply:
I understand that but I am just providing an easy way to apply it the YAML in case they need to apply it. Would it be a stretch to assume that whoever is using this script would have `kubectl` installed? This is for Kubernetes after all, and Kubernetes works for Windows as well as far as I understand?

Reply:
We are asking the user to apply the YAML manifest, which could totally be done using the Python client without using ```kubectl```.

```cat``` is not a valid Windows command I believe.